### DISCUSSION TITLE
CycloneDX spec v1.4 support

### DISCUSSION THREAD

**Discussion by kevinlinglesas** [rad-security/kbom#102] [Q&A] [answered]
Is there any support for v1.4 cyclonedx spec or does kbom only support v1.5?

**Comment by mateuszdyminski** [comment, marked as answer]
Hi @kevinlinglesas, thanks for reaching us out!

We regret to inform you that our KBOM tool currently does not support CycloneDX version 1.4. The decision to support CycloneDX version 1.5 was made because we leverage specific fields introduced in this version(`cyclonedx.ComponentTypePlatform`) See: https://github.com/CycloneDX/cyclonedx-go/commit/eb041b55b2eb8685a37be6f7a9c265fb6528377b

Unfortunately, this means that compatibility with CycloneDX version 1.4 is not feasible at this time.
